src/main/sensors/esc_sensor.c

```diff
@@ -361,8 +361,10 @@ void escSensorProcess(timeUs_t currentTimeUs)
     }
 }
 
+#endif
```

Comment:
`calcEscRpm` is technically in wrong file (and placing #endif here is ugly).
Maybe it can be renamed to  'erpmToRpm' or similar and moved to some motor-related file.

Reply:
Agreed. Please, could you propose what file this should be moved to?

Reply:
Would prefer `dshot.c`

Reply:
Reworked